examples/plotting/file/slider_callback_policy.py

```diff
@@ -1,3 +1,12 @@
+''' An plot of two interactive sliders. The values are updated simultaneously as the slider bars are dragged to different values.
+This example demonstrates how ``CustomJS`` callbacks react to user interaction events.
```

Comment:
I missed this earlier, it would also be preferred t shorten these lines, then I think the PR is ready to merge

Reply:
Oh right! Let me fix this quickly.